npc/re/other/LapineEnchants.txt

```diff
@@ -0,0 +1,2097 @@
+//===== rAthena Script =======================================
```

Comment:
itens ID (15353, 15354, 15346, 15347, 15278, 15279, )
F_Lapine_Gemstone_Of_Time [BUG]
option 1
RDMOPT_HEAL_VALUE; set getarg(1), rand(3,20); } must be 1~5
RDMOPT_RANGE_ATTACK_DAMAGE_TARGET; set getarg(1), rand(3,20); } must be 1~5
RDMOPT_DEC_SPELL_DELAY_TIME; set getarg(1), rand(3,20); } must be 1~5
option 2
RDMOPT_HEAL_VALUE; set getarg(3), rand(3,20); } must be 1~5
(Check all elements) RDMOPT_BODY_ATTR_WIND; set getarg(3), 1; }


itens ID (20842, 20843, 20844)
F_Lapine_Candy_Pouch_Blessing_Scroll_R
F_Lapine_Candy_Pouch_Blessing_Scroll_M
F_Lapine_Candy_Pouch_Blessing_Scroll_W [BUG]
option 1
RDMOPT_DEC_SPELL_DELAY_TIME; set getarg(1), rand(1,5); } must be 1~3


F_Lapine_Magic_Cat_Hand [BUG]
option 1
must be de 5~20
	if (.@r < 90)  { set getarg(0), RDMOPT_VAR_ATTPOWER; set getarg(1), rand(1,20); }
	if (.@r < 100) { set getarg(0), RDMOPT_VAR_ATTMPOWER; set getarg(1), rand(1,20); }



F_Lapine_Grown_Gunslinger [BUG]

option 2
RDMOPT_HEAL_MODIFY_PERCENT; set getarg(3), rand(1,7); } must be 5~15


F_Lapine_Fan_Modification_Kits [BUG]
option 1
RDMOPT_VAR_MAXHPPERCENT; set getarg(1), rand(1,1); } must be 1~3
option 2
RDMOPT_VAR_MAXHPPERCENT; set getarg(3), rand(1,1); } must be 1~3

F_Lapine_Uniform_Repair_Kits [BUG]
Change to ", 1;"
		 if (.@r < 10)  { set getarg(0), RDMOPT_BODY_ATTR_WATER; set getarg(1), rand(1,3); }
	else if (.@r < 20)  { set getarg(0), RDMOPT_BODY_ATTR_GROUND; set getarg(1), rand(1,3); }
	else if (.@r < 30)  { set getarg(0), RDMOPT_BODY_ATTR_FIRE; set getarg(1), rand(1,3); }
	else if (.@r < 40)  { set getarg(0), RDMOPT_BODY_ATTR_WIND; set getarg(1), rand(1,3); }
	else if (.@r < 50)  { set getarg(0), RDMOPT_BODY_ATTR_SAINT; set getarg(1), rand(1,3); }
	else if (.@r < 60)  { set getarg(0), RDMOPT_BODY_ATTR_DARKNESS; set getarg(1), rand(1,3); }